### PR TITLE
try to fix GoLilyPad servers

### PR DESCRIPTION
**Pull request by creeper123123321**
ViaVersion/ViaFabric#21

### CODE REVIEW

File: common/src/main/java/com/viaversion/viaversion/protocols/protocol1_9to1_8/providers/CompressionProvider.java [hunk 1]
```diff
@@ -22,7 +22,7 @@ public void handlePlayCompression(UserConnection user, int threshold) {
 
         ChannelPipeline pipe = user.getChannel().pipeline();
 
-        if (threshold == -1) {
+        if (threshold <= 0) {
```
Comment (RaphiMC) [inline common/src/main/java/com/viaversion/viaversion/protocols/protocol1_9to1_8/providers/CompressionProvider.java:25]: 0 is a valid threshold (means all packets are compressed)

Comment (creeper123123321) [reply common/src/main/java/com/viaversion/viaversion/protocols/protocol1_9to1_8/providers/CompressionProvider.java:25]: wiki vg says 0 is uncompressed

Comment (RaphiMC) [reply common/src/main/java/com/viaversion/viaversion/protocols/protocol1_9to1_8/providers/CompressionProvider.java:25]: Minecraft Client code: 0 seems valid
grafik

**Comment by RaphiMC** [comment]
Looks good to me now

**Review by RaphiMC — APPROVED** [review]
Looks good to me now

**Review by kennytv — APPROVED** [review]

**Pull request merged**